Cypress/cypress/integration/smoke/smoke.js

```diff
@@ -0,0 +1,102 @@
+
+/**
+ * This test will go through all the core user stories
+ * and try to do everything a user would do
+ */
+
+import * as user from "../../fixtures/Users.json";
+import * as workflows from "../../fixtures/Workflows.json";
+
+const setup = () => {    
+    cy.requestLogin(user.AdminName, user.AdminPassword);    
+    cy.waitForCluster("Self-Agent");    
```

Comment:
You may use the waitForCluster() separately according to scenarios, as we don't have to wait for self-agent, when we test account settings.

Reply:
Sure, have added a conditional check for the same